CHANGELOG.md

```diff
@@ -9,8 +9,14 @@ can be found at [#317](https://github.com/grafana/agent/issues/317).
   ARMv6 docker builds are also now available.
   (@rfratto)
 
+- [ENHANCEMENT] A sigv4 install script for Prometheus has been added. (@rfratto)
```

Comment:
Is it common to refer the AWS Prom offering as "sigv4"? Are we expecting users to understand that this means talking to AWS managed prom?

Reply:
I've been referring to it as sigv4 generically, I don't think supporting SigV4 is always going to be exclusive to AMP, so I want to keep this open. 

(Also, we have way too many install scripts and we need to start consolidating with one Helm chart)